client/my-sites/plans/main.jsx

```diff
@@ -143,11 +143,14 @@ class Plans extends Component {
 			);
 		}
 
+		const hideFreePlan =
+			! isEnabled( 'onboarding/2023-pricing-grid' ) || currentPlan.productSlug !== 'free_plan';
```

Comment:
@chriskmnds This might overlap with your work on https://github.com/Automattic/martech/issues/1450 - this allows the Free plan to reach the new Plans and Plan Comparison Grids, if it's the current plan.

Reply:
Thanks for mentioning @claudiucelfilip . Indeed.

I guess we are bound to reach these overlaps if some of us are working on general open-ended tickets like the one you linked to from the description.

> this allows the Free plan to reach the new Plans and Plan Comparison Grids, if it's the current plan

It's not visible to me where this has been decided. I've been trying to figure this out in Slack today. I understand this is part of design/intention? 🤔